integration-cli/docker_api_logs_test.go

```diff
@@ -172,6 +172,8 @@ func (s *DockerSuite) TestLogsAPIUntil(c *check.C) {
 
 	// Get timestamp of second log line
 	allLogs := extractBody(c, types.ContainerLogsOptions{Timestamps: true, ShowStdout: true})
+	c.Assert(len(allLogs), checker.GreaterOrEqualThan, 3)
```

Comment:
Looks like the logs return an empty `""` as last entry, so switched to `checker.GreaterOrEqualThan` instead of `checker.Equal`